styles.css

```diff
@@ -0,0 +1,137 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  margin: 10px;
+  padding: 10px;
+}
+
+.nav-menu-content {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.nav-menu {
+  list-style: none;
+  display: flex;
+  gap: 60px;
+}
+
+a:hover {
+  background: #acb7c3;
+  transition: 0.4s;
+}
+
+.logo {
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: bold;
+  color: #6070ff;
+}
+
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+@media (max-width: 768px) {
+  .hamburger {
+    display: block;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 70px;
+    gap: 0;
+    flex-direction: column;
+    background-color: #6070ff;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+  }
+
+  .nav-items {
+    margin: 16px 0;
+  }
+
+  .nav-menu.active {
+    left: 0;
+  }
+}
+
+.homepage {
+  background: url("Image/backimg.png");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  height: 718px;
+}
```

Comment:
- [ ] Great job fixing what the previous reviewer requested, however, your page is different from the design on Figma. Kindly fix that.
- [ ] Please make sure that you use the same font, spacing, color specified in the design.
Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154070487-d80badd7-107e-4efe-80c0-d46a117af966.png) | ![image](https://user-images.githubusercontent.com/59416136/154070707-18c5f3fd-ed17-40a8-87b7-4dce79d53ca3.png)

Reply:
fixed background not matching from desktop image to mobile image, fixed alignment issue, updated linked in image and fixed index and styles codes for it.